test/sol6/kyberDao.js

```diff
@@ -1042,7 +1042,7 @@ contract('KyberDAO', function(accounts) {
           1, startBlock + 2, startBlock + 2 + minCampPeriod,
           minPercentageInPrecision, cInPrecision, tInPrecision, [1, 2, 3], '0x', {from: daoOperator}
         ),
-        "newCampaign: already had network fee for this epoch"
+        "validateParams: already had network fee for this epoch"
```

Comment:
I think can write
already had network fee campaign for ...

Reply:
Updated